components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -114,21 +109,21 @@ export const PrebuildDetailPage: FC = () => {
         return () => {
             disposable.dispose();
         };
-    }, [prebuildId, currentPrebuild?.status?.phase?.name]);
```

Comment:
this dependency made us re-watch the status whenever the prebuild's phase changed